src/game/services/Lookup/index.ts

```diff
@@ -28,4 +28,31 @@ export class Lookup {
 
     return card
   }
+
+  /**
+   * Returns all the IDs for players that are not the current user's.
+   */
+  getOpponentPlayerIds = (game: IGame) => {
+    const playerIds = Object.keys(game.table.players)
+
+    const nonUserPlayers = playerIds.filter(
```

Comment:
minor naming nitpick... based on the method name, how about naming this var `const opponentPlayerIds =`?

Reply:
Good catch! This name was a holdover from an earlier iteration. Changed to `opponentPlayerIds` in https://github.com/jeremyckahn/farmhand-shuffle/commit/b1a3b95dea0fd64accbc17113e31c34bcdb0b86c!